rootfs/etc/nginx/lua/balancer/sticky.lua

```diff
@@ -15,13 +15,10 @@ function _M.new(self, backend)
   local o = {
     instance = self.factory:new(nodes),
     cookie_name = backend["sessionAffinityConfig"]["cookieSessionAffinity"]["name"] or "route",
```

Comment:
Why are you keeping this?

Reply:
Thought it looked cleaner than `or`ing it with `"route"` in 2 different places. But this is probably more confusing, because everything else is in `cookie_sesstion_affinity`. fixed.